Makefile

```diff
@@ -254,6 +254,14 @@ helm-install-redis: helm-bitnami
 helm-install-prometheus: helm-bitnami
```

Comment:
Could we remove this as part of this PR?

Reply:
Sure